ports/blaze/portfile.cmake

```
@@ -1,25 +1,22 @@
 vcpkg_from_bitbucket(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO blaze-lib/blaze
-    REF 0380370f0626de2ad0ec7ea815803e22bbf6b42e
-    SHA512 47df2a291febf0565787ed9c4222ee1958d03c5b76c54923e4d8e44f75321c266e81d69ba4512a4ab07d7e431c065d025e01269cf9c1d5f0927f133885def4c7
+    REF v3.8.1
+    SHA512 6dfd3cb46d796b94cc44a30c4cd5ebfb366d2eb312d75a28787cacb4636df52e4e4e3dce3d9501bf2c07e7fd3621e8ce7f9ffa61a950a4146375b12d75d4872b
     HEAD_REF master
-    PATCHES
-        fix-vm-build.patch
 )
 
-vcpkg_configure_cmake(
-    SOURCE_PATH ${SOURCE_PATH}
-    PREFER_NINJA
+vcpkg_cmake_configure(
+    SOURCE_PATH "${SOURCE_PATH}"
     OPTIONS
         -DBLAZE_SMP_THREADS=OpenMP
 )
 
-vcpkg_install_cmake()
+vcpkg_cmake_install()
 
-vcpkg_fixup_cmake_targets(CONFIG_PATH share/blaze/cmake)
+vcpkg_cmake_config_fixup()
```

I think this still needs to be
```suggestion
vcpkg_cmake_config_fixup(CONFIG_PATH share/blaze/cmake)
```
otherwise the files are produced into `share/blaze/cmake/blaze-config.cmake`.